src/crypto/tls/conn.go

```diff
@@ -95,6 +95,9 @@ type Conn struct {
 	// clientProtocol is the negotiated ALPN protocol.
 	clientProtocol string
 
+	// selectedGroup is the negotiated group
```

Comment:
This parameter is only consumed by tests in cfkem_test.go. Is there any way to test the same thing without needing to make this change? This would help reduce maintenance cost. Some potential options:
- ConnectionState.CipherSuite (not sure how we compute the cipher suite when the new code point is negotiated)
- ClientHelloInfo.SupportedCurves

Reply:
We will need this for metrics later anyway.

Reply:
Changed it to use CFEvent API.